Px.Utils/ModelBuilders/ValueParserUtilities.cs

```diff
@@ -56,26 +56,36 @@ public static TimeDimensionInterval ParseTimeIntervalFromTimeVal(string input, P
         /// List of value strings excluding the interval part.
         /// If the input string is in the range format, empty list is returned.
         /// </returns>
-        /// <exception cref="ArgumentException">If the input string does not match the expected timeval format.</exception>
         public static List<string> GetTimeValValueList(string input, PxFileConfiguration? conf = null)
         {
             conf ??= PxFileConfiguration.Default;
-
-            if (input.StartsWith(conf.Tokens.Time.TimeIntervalIndicator, StringComparison.InvariantCulture))
+            int endOftoken = input.IndexOf(conf.Symbols.Value.TimeSeriesIntervalEnd);
+            int firtsStringDelimeter = input.IndexOf(conf.Symbols.Value.StringDelimeter, endOftoken);
+            if (firtsStringDelimeter >= 0)
             {
-                int endOftoken = input.IndexOf(conf.Symbols.Value.TimeSeriesIntervalEnd);
-                int firtsStringDelimeter = input.IndexOf(conf.Symbols.Value.StringDelimeter, endOftoken);
-                if (firtsStringDelimeter >= 0)
-                {
-                    return input[firtsStringDelimeter..]
-                        .SplitToListOfStrings(conf.Symbols.Value.ListSeparator, conf.Symbols.Value.StringDelimeter);
-                }
-                else return [];
+                return input[firtsStringDelimeter..]
+                    .SplitToListOfStrings(conf.Symbols.Value.ListSeparator, conf.Symbols.Value.StringDelimeter);
             }
-            else
+            else return [];
+        }
+
+        /// <summary>
+        /// Removes the interval entry from the beginning of a time value string
+        /// and returns the rest as a string.
+        /// </summary>
+        /// <param name="input">The complete timeval value in one language.</param>
+        /// <param name="conf">Configuration used for parsing the value strings and the interval part.</param>
+        /// <returns> Value string excluding the interval part. If the string is not enclosed in string delimeters, empty string is returned.</returns>
+        public static string GetTimeValValueString(string input, PxFileConfiguration? conf = null)
```

Comment:
Change this to only parse the range strings and rename accordingly.

Reply:
Renamed and added functionality